docs/WORKSPACE_SECURITY.md

```diff
@@ -0,0 +1,80 @@
+# Workspace File Security Configuration
```

Comment:
Move this documentation to the workspace documentation page in the docs (create one if needed)